palette/benchmarkgo17_test.go

```diff
@@ -0,0 +1,74 @@
+// Copyright ©2016 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build go1.7
+
+package palette
+
+import (
+	"fmt"
+	"image/color"
+	"math/rand"
+	"testing"
+)
+
+func BenchmarkColorMap_At(b *testing.B) {
```

Comment:
I meant "informative", though border line; I would have been extremely surprised if they had fallen the other way.

I don't see why plotting is going to end up in an inner loop, at least not under the normal understanding of that phrase; there are too many things in plot that take too long (with justification) for that to be a sensible engineering solution.